src/NuGetGallery/Areas/Admin/Controllers/FeaturesController.cs

```diff
@@ -196,15 +197,15 @@ private async Task<string> TrySaveFlags(FeatureFlagsViewModel model, string cont
 
             switch (result)
             {
-                case FeatureFlagSaveResult.Ok:
+                case ContentSaveResult.Ok:
                     // The flags have been persisted. Refresh this instance's cache immediately.
                     await _cache.RefreshAsync();
 
                     var refreshSeconds = _config.FeatureFlagsRefreshInterval.TotalSeconds;
```

Comment:
nit: should we rename this to maintain consistency with the FeatureFlagSaveResult > ContentSaveResult change?

```suggestion
                    var refreshSeconds = _config.ContentRefreshInterval.TotalSeconds;
```

Reply:
probably keep as `FeatureFlagsRefreshInterval` since it only works in feature flag now